mod/provider/s3.js

```diff
@@ -1,102 +0,0 @@
-/**
```

Comment:
I think let's also reroute the user with a deprecation warning for this endpoint.

Reply:
Addressed in latest push 👍